widgets/views/breakingNews.php

```diff
@@ -3,14 +3,8 @@
 use yii\helpers\Url;
 ?>
 <script type="text/javascript">
```

Comment:
@yurabakhtin  On inline Script we need to ensure there is a `Nonce()` included.

You can either add the nonce to the script tag <script <?=Html::nonce() ?> .....   oder generate the script tag <?= Html::beginSCript

Reply:
Fyi, the nonce() protection is not enabled by default.

Reply:
@luke- ok, fixed in commit https://github.com/humhub-contrib/breakingnews/pull/14/commits/8e450f4024728f16c06a1d5e8f9d4f7918e3c4f9.